tests/loaders/test_configloader.py

```diff
@@ -95,6 +95,30 @@ def test_config_loader_load_mapping_yolox(self, configloader):
         assert test_map["clock"] == 74
         assert test_map["toothbrush"] == 79
 
+    def test_config_loader_obj_det_wildcard_efficientdet(self, configloader):
+        """Test object detection wildcard for EfficientDet models"""
+        node_name = "model.efficientdet"
+        key = "detect_ids"
+        val = ["*"]
+        # fmt: off
+        ground_truth = ("detect_ids", [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 26, 27, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 66, 69, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 83, 84, 85, 86, 87, 88, 89])
+        # fmt: on
```

Comment:
I understand without the magic comments this will get formatted to 80+ lines tall, but is it possible to still manually format so what it still wraps within 90 chars or so? Maybe something like
```
ground_truth = (
    "detect_ids",
    [
        0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 14, 15, 16, 17, 18,
        19, 20, 21, 22, 23, 24, 26, 27, 30, 31, 32, 33, 34, 35, 36, 37,
        38, 39, 40, 41, 42, 43, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54,
        55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 66, 69, 71, 72, 73, 74,
        75, 76, 77, 78, 79, 80, 81, 83, 84, 85, 86, 87, 88, 89
    ]
)
```
So that it will be easier to edit if we ever need to in the future. Similar for the L116 below

Reply:
Good idea: reformatted lines.